### PR TITLE
Add id to expected json

### PR DESCRIPTION
**Pull request by shruacha1234**
Signed-off-by: Shruthi <Shruthi.Acharya@ibm.com>

### CODE REVIEW

**Comment by chandrams** [comment]
@shruacha1234 - Can you please resolve the conflicts.

**Comment by shruacha1234** [comment]
> @shruacha1234 - Can you please resolve the conflicts.

@chandrams done

File: tests/scripts/common_functions.sh [hunk 1]
```diff
@@ -772,6 +772,7 @@ function create_expected_listapptunables_json() {
 					printf '\n }], \n' >> ${file_name}
 				fi
 			done
+			printf '\n  "id": '$(cat ${json_file} | jq 'sort_by(.application_name)' | jq '.['${count}'].layers[].id')',' >> ${file_name}
```
Comment (chandrams) [inline tests/scripts/common_functions.sh:775]: id needs to be added to expected search space json too

**Review by chandrams — APPROVED** [review]
lgtm

**Pull request merged**